flight/Modules/VtolPathFollower/vtol_follower_fsm.c

```diff
@@ -142,8 +129,7 @@ static void hold_position(float north, float east, float down);
 
 /**
  * The state machine for landing at home does the following:
- * 1. enable holding at the current location
- * 2  TODO: if it leaves the hold region enable a nav mode
+ * enable holding at the current location
```

Comment:
outside of a certain range the endpoint control doesn't produce as good results as the path navigation. you could also rephrase it as the endpoint controller itself being more intelligent for large errors